src/main/java/com/powsybl/openloadflow/sensi/WoodburyEngine.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.sensi;
+
+import com.powsybl.math.matrix.DenseMatrix;
+import com.powsybl.math.matrix.LUDecomposition;
+import com.powsybl.openloadflow.dc.DcLoadFlowContext;
+import com.powsybl.openloadflow.dc.equations.AbstractClosedBranchDcFlowEquationTerm;
+import com.powsybl.openloadflow.dc.equations.ClosedBranchSide1DcFlowEquationTerm;
+import com.powsybl.openloadflow.dc.equations.DcEquationSystemCreationParameters;
+import com.powsybl.openloadflow.network.LfBranch;
+import com.powsybl.openloadflow.network.PiModel;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
+ * @author Gaël Macherel {@literal <gael.macherel@artelys.com>}
+ */
+public class WoodburyEngine {
+
+    final DcLoadFlowContext loadFlowContext;
```

Comment:
only usage of `DcLoadFlowContext` is in `calculatePower`, and actually only using the `DcEquationSystemCreationParameters` 

isn't actually the `DcEquationSystemCreationParameters` that you need in the WoodburyEngine ?

or are there more future developments to come where you will need really the `DcLoadFlowContext` ?

Reply:
Yes you are right. I have switched the whole DcLoadFlowContext to only the DcEquationSystemCreationParameters.